build.gradle

```diff
@@ -150,6 +150,16 @@ project(':iceberg-data') {
   }
 }
 
+project(':examples') {
```

Comment:
Do we envision a common examples module for all query engines or one for every query engine?